tests/test-http-api/test-config.js

```diff
@@ -143,6 +148,69 @@ describe('config', () => {
         done()
       })
     })
+
+    describe('/config/replace', () => {
+      it('returns 400 if no config is provided', (done) => {
+        const form = new FormData()
+        const headers = form.getHeaders()
+
+        streamToPromise(form).then(payload => {
```

Comment:
Nope, [this project is still on standard 5](https://github.com/ipfs/js-ipfs/blob/master/package.json#L63), and we have those all over the place...  If you want, I can update standard and make all the changes on another PR, so we don't pollute this one :octocat: